### DISCUSSION TITLE
이번에 nextjs 를 사용해보려고 하는데 14 버전 어떻게 생각하시나요

### DISCUSSION THREAD

**Discussion by HJ97Kim** [MoCo-v2/moco-client#5] [Q&A] [answered]
어떻게 생각하시나요

**Comment by wlswo** [comment, marked as answer]
이번에 14버전 업데이트로 최대 53% 속도 향상 된 로컬 서버 구동 속도 및 여러가지 이점이 있다고 생각합니다.
물론 기존에 사용중인 패키지와 종속성 문제가 있을수도 있지만 한번 사용해보시는게 좋을거 같아요
